{"name": "test_premise[login0-case_data0]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/Users/nali/AutoApiTest/config'", "trace": "request = <SubRequest 'login' for <Function test_premise[login0-case_data0]>>\n\n>   ???\n\n/Users/nali/AutoApiTest/conftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/Users/nali/ApiAutoTest/common/unit/api_send_check.py:28: in api_send_check\n    res = api_send.api_send(case_data, case_dict, case_path, relevance)\n/Users/nali/ApiAutoTest/common/unit/api_send.py:85: in api_send\n    host = conf_manage(cur_host, 'HOST')\n/Users/nali/ApiAutoTest/common/config_module/config_manage.py:22: in conf_manage\n    param = cf.read_config(name)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <common.config_module.config_read.Config object at 0x11dbf4c88>\nsection = 'HOST'\n\n    def read_config(self, section):\n        \"\"\"\n        以dict方式返回ini文件中option键值对\n        :param section: section\n        :return: option键值对\n        \"\"\"\n        cf = ConfigOveride()\n>       os.chdir(self.path)\nE       FileNotFoundError: [Errno 2] No such file or directory: '/Users/nali/AutoApiTest/config'\n\n/Users/nali/ApiAutoTest/common/config_module/config_read.py:84: FileNotFoundError"}, "parameters": [{"name": "login", "value": "{'test_name': 'login_h5', 'info': '密码登录m站', 'is_run': None, 'host': '${HOST1}$', 'headers': '${headers1}$', 'address': '/passport/api/pwdLogin', 'http_type': 'https', 'request_type': 'post_cookie', 'parameter_type': 'data', 'parameter': {'passport': 17521200974, 'password': 'anqi666', 'passportType': 1, 'sign': 'F017A0CD674A8B7A4F52B56898D78E6A'}, 'timeout': 10, 'cookies': False, 'check': {'expected_result': None, 'check_type': 'no_check', 'expected_code': None}, 'file': False, 'global_relevance': None, 'relevance': None}"}, {"name": "case_data", "value": "{'is_run': False}"}], "start": 1617801448718, "stop": 1617801448718, "uuid": "c9cf34fc-1530-4d8b-8ad3-0205dcb70f60", "historyId": "24cea6b14171dd58eb287b495addd0b3", "testCaseId": "f56645c48f56e52a5bc144f4fc40aafb", "fullName": "tests.TestScript.countApi.test_count.TestCount#test_premise", "labels": [{"name": "feature", "value": "count"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "@pytest.mark.skipif(False, reason='该用例的用例等级为：blocker，不在本次运行级别['blocker', 'critical', 'normal', 'minor']内')"}, {"name": "parentSuite", "value": "tests.TestScript.countApi"}, {"name": "suite", "value": "test_count"}, {"name": "subSuite", "value": "TestCount"}, {"name": "host", "value": "anqidediannao.local"}, {"name": "thread", "value": "86863-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.TestScript.countApi.test_count"}]}